### PR TITLE
Issue 9 python source file name not matched

### PR DESCRIPTION
**Pull request by sarah-afiya**
Issue #9

### CODE REVIEW

File: Summary/analyze/MethodLinker/matcher.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def get_source_files(repo_name, repo_path):
     # Only look at the top level directory in this loop
     for root, dirs, files in os.walk(repo_path):
         for dir_name in dirs:
-            if dir_name.lower() == "src" or dir_name == repo_name:
+            if dir_name.lower() == "src" or dir_name.lower() == repo_name.lower():
```
Comment (snadi) [inline Summary/analyze/MethodLinker/matcher.py:90]: what was the example that failed before?

**Comment by snadi** [comment]
Please mention the issue in the description so it is directly linkable (e.g., Issue #9 )

File: Summary/analyze/MethodLinker/matcher.py (hunk 1 above)
Comment (snadi) [reply Summary/analyze/MethodLinker/matcher.py:90]: in the documented example, the directory name and repo name were both `requests` so there was no mismatch in capitalization or am I missing something?

Comment (sarah-afiya) [reply Summary/analyze/MethodLinker/matcher.py:90]: The library name is "Requests" but in the repo, the directory name is "requests"

**Review by snadi — APPROVED** [review]

**Pull request merged**